_episodes/06-control-flow.md

```diff
@@ -83,6 +108,23 @@ Therefore she restarts her REPL.
 > {: .solution}
 {: .challenge}
 
+With the new `Trebuchet` defined with a complete `AbstractArray` interface,
+Melissa tries again to modify a counterweight by index:
+
+~~~
+trebuchet = Trebuchet(500, 0.25pi)
+trebuchet[1] = 2
+print(trebuchet)
```

Comment:
```suggestion
trebuchet
```

Reply:
These three lines of input map to the three items of output in the following block. `print(trebuchet)` does not return `trebuchet`, it gives the two-item list of its elements, in this case `[2.0, 0.7853981633974483]`.